user/src/main/resources/application.yml

```
@@ -84,4 +84,8 @@ kakao:
 logging:
   pattern:
     console: '%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} [%-0.32X{traceId:-},%-0.16X{spanId:-}] %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:-}){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:%wEx}'
-    # ✅ 모든 콘솔 로그에 traceId, spanId 추가
\ No newline at end of file
+    # ✅ 모든 콘솔 로그에 traceId, spanId 추가
+
+jwt:
+  secret:
+    key: ${JWT_SECRET_KEY}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)